src/components/Calculator.js

```diff
@@ -1,38 +1,54 @@
-          <button type="submit" className="input-btn" value="number">5 </button>
-          <button type="submit" className="input-btn" value="number">6 </button>
-          <button type="submit" className="input-btn btn-org" value="number">- </button>
-          <button type="submit" className="input-btn" value="number">1 </button>
-          <button type="submit" className="input-btn" value="number">2 </button>
-          <button type="submit" className="input-btn" value="number">3 </button>
-          <button type="submit" className="input-btn btn-org" value="number">+ </button>
-          <button type="submit" className="input-btn btn-zero" value="number">0 </button>
-          <button type="submit" className="input-btn" value="number">. </button>
-          <button type="submit" className="input-btn btn-org" value="number">= </button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">AC</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">+/-</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">%</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">÷</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">7</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">8</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">9</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">x</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">4</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">5</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">6</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">-</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">1</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">2</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">3</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">+</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn btn-zero" value="number">0</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">.</button>
+          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">=</button>
```

Comment:
- [ ] Please know that you should be using the principle DRY here so you don't repeat yourself much, I recommend that you create a Button component and use what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.